index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+    <header class="nav">
+        <a href="" class="m-logo" >My Logo</a>
+         <nav>
+           <ul class="header-link">
+             <li class="link-right">
+                <a href="/someurl" class="inline"><b>Portfolio</b></a>
+                <a href="/someurl" class="inline"><b>About</b></a>
+                <a href="/someurl" class="inline"><b>Contact Us</b></a>
+             </li>
+           </ul>
+        </nav>
+    </header>
+    
+
+
+
+     <!-- header images section -->
+    <section class="abl">
+  
+    </section>
+
+    <!-- Header images section closed here -->
+       <!-- Section 2 started -->
+       <section class="section2">
+        <h2 class="h23">I’m Aziz Glad to see you!</h2>
+         <h4  class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+  <h5 class="ets" >ETS CONNECT</h5>
+ <ul class="social-menue">
+  <li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+  <li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+  <li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+ </ul>
+          
+ </section>
```

Comment:
Kindly make this section resemble Figma design as closely as possible for both mobile and desktop 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187712557-c4360dcd-5f60-4f3e-aab8-545fd3a60d57.png)
![image](https://user-images.githubusercontent.com/73714615/187712611-5a153581-f9a7-4100-b60e-54b564cf485e.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187712687-db813e7a-22c3-4a9c-b1c8-6996fb778e86.png)
![image](https://user-images.githubusercontent.com/73714615/187712749-e84b4374-1f74-44cb-8d9c-4a8b68570d21.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉